tests/tmt/plans/main.fmf

```diff
@@ -0,0 +1,35 @@
+---
+# This prepare is used to control when the ignition is installed using
+# the distribution package or when it is built from source in the test environment
+prepare:
+  - name: Set IGNITION_BIN_DIR when built from source
+    when: use_built_form_src is defined and use_built_form_src == true
```

Comment:
global change to `use_built_from_src` -- no more comments about it from now to the end of the review